extensions/wikia/PremiumPageHeader/PremiumPageHeaderController.class.php

```diff
@@ -41,22 +320,38 @@ private function getExplore() {
 		}, $explore );
 
 		return [
-			'text' => wfMessage( 'pph-explore' )->escaped(),
+			'text' => wfMessage( 'pph-explore' )->inContentLanguage()->escaped(),
 			'children' => array_filter( $children, function ( $child ) {
 				return !empty( $child );
 			} )
 		];
 	}
 
-	private function getDiscuss() {
+	private function getDiscuss(): array {
 		global $wgEnableDiscussionsNavigation, $wgEnableDiscussions, $wgEnableForumExt;
 
-		$href = !empty( $wgEnableDiscussionsNavigation ) && !empty( $wgEnableDiscussions ) && empty( $wgEnableForumExt ) ?
-			'/d' : Title::newFromText( 'Forum', NS_SPECIAL )->getLocalURL();
+		$href =
+			!empty( $wgEnableDiscussionsNavigation ) && !empty( $wgEnableDiscussions ) &&
```

Comment:
not sure if it is worth to do it now, but we should create separate variable for checking if discuss button should point to discussions or forum as this is exactly the same condition like https://github.com/Wikia/config/blob/c8e54b44bb779d9348af3374413933110e0c1c22/CommonExtensions.php#L1278-L1278